backend/ticketvise/views/inbox/users.py

```diff
@@ -17,7 +17,7 @@ class InboxUsersView(InboxCoordinatorRequiredMixin, TemplateView):
     def get_context_data(self, **kwargs):
         inbox = get_object_or_404(Inbox, pk=self.kwargs.get("pk"))
 
-        if not self.request.user.is_coordinator_for_inbox(inbox):
+        if not self.request.user.is_coordinator_for_inbox(inbox) and not self.request.user.is_superuser:
```

Comment:
Ik zie vaak dat er "self.request.user.is_superuser" wordt gecheckt, kan dit neit beter in de is_coordinator_for_inbox etc worden gedaan. Dan weten we zeker dat het overal werkt.